scalafmt-core/shared/src/main/scala/org/scalafmt/internal/FormatOps.scala

```diff
@@ -2073,14 +2073,16 @@ class FormatOps(
           style: ScalafmtConfig
       ): Option[OptionalBracesRegion] =
         ft.meta.leftOwner match {
-          case ParamClauseParent(t @ Defn.ExtensionGroup(_, _, b: Term.Block))
-              if isBlockStart(b, nft) =>
+          case ParamClauseParent(
+                t @ Defn.ExtensionGroup.internal.Impl(_, b: Term.Block)
+              ) if isBlockStart(b, nft) =>
```

Comment:
```suggestion
          case ParamClauseParent(
                t : Defn.ExtensionGroup
              ) if isBlockStart(t.body, nft) =>
```
I would avoid using the internal Impl classes since they might not be binary compatible.